draft-selander-ace-ake-authz.md

```diff
@@ -476,142 +478,37 @@ V receives the voucher response from W over the secure connection. If the receiv
 
 V verifies CRED_I and that U is an admissible device and then continues the EDHOC processing, or else discontinues the protocol.
 
-# ACE Profile
+# REST Interface at W
 
-The messages specified in this document may be carried between the endpoints in various protocols. This section defines an embedding as a profile of the ACE framework (see Appendix C of {{I-D.ietf-ace-oauth-authz}}).
+The interaction between V and W is enabled through a RESTful HTTPS interface exposed by W.
+V MUST perform a TLS handshake with W, reachable at the URI specified in LOC_W.
+V MUST use the TLS client authentication to authenticate to W, using the certificate containing the PK_V public key.
 
-* U plays the role of the ACE Resource Server (RS).
+## HTTP URIs
 
-* V plays the role of the ACE Client (C).
+W MUST support the use of the path-prefix "/.well-known/", as defined in {{RFC5785}}, and the registered name "ake-authz".
+A valid URI thus begins with "https://www.example.com/.well-known/ake-authz".
+Each operation specified in the following is indicated by a path-suffix.
+Since payloads carry CBOR binary strings, they MUST be base64-encoded, as specified in Section 4 of {{RFC4648}}.
 
-* W plays the role of the ACE Authorization Server (AS).
+## Voucher Request (/voucherrequest)
 
-Many readers who are used to the diagram having the Client on the left may be surprised at the cast of characters.
-The "resource" which C (V) is trying to access is the "ownership" of U.
-The AS (W) is the manufacturer (or previous owner) of RS (U), and is therefore in a position to grant C (V) ownership of RS (U).
+To request a voucher, V MUST issue an HTTP request:
 
-C and RS use EDHOC's EAD to communicate.
-C and RS use the EDHOC protocol to protect their communication.
-EDHOC also provides mutual authentication of C and RS, assisted by the AS.
+* Method is POST
+* Payload is the base64-encoded serialization of the Voucher Request object, as specified in {{voucher_request}}.
 
-## Protocol Overview
+In case of successful processing at W, W MUST issue a 200 OK response with payload containing base64-encoded Voucher Response object, as specified in {{voucher_response}}.
```

Comment:
why is it base64 encoded?

Reply:
I thought it was necessary but it seems I am wrong. Does the rest of the PR look good? Anything else that needs to be specified?

Reply:
I am not sure if there is anything else, but merge it.

Reply:
fixed